### PR TITLE
docs(howto): add custom domain to functions MTA-1777

### PR DESCRIPTION
**Pull request by bene2k1**

### CODE REVIEW

File: compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+meta:
+  title: How to add a custom domain to a function
+  description: This page explains how toadd a custom domain to a function
```
Comment (RoRoJ) [inline compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx:4]:
```suggestion
  description: This page explains how to add a custom domain to a function
```

File: compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+content:
+  h1: How toadd a custom domain to a function
```
Comment (RoRoJ) [inline compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx:6]:
```suggestion
  h1: How to add a custom domain to a function
```

File: compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  paragraph: This page explains how to add a custom domain to a function
+dates:
+  validation: 2022-02-21
+  posted: 2022-02-21
+---
+
+Each function comes with an automatically generated domain on which it is available. However it is possible to use your own domain name to address your functions to facilitate their use.
```
Comment (RoRoJ) [inline compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx:13]:
```suggestion
Each function comes with an automatically generated domain on which it is available. However it is possible to assign your own domain name to make it easier to use your functions.
```

File: compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+- A running [Serverless Function](https://www.scaleway.com/en/docs/compute/functions/how-to/deploy-a-function/)
```
Comment (RoRoJ) [inline compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx:18]:
```suggestion
- You have a running [Serverless Function](https://www.scaleway.com/en/docs/compute/functions/how-to/deploy-a-function/)
```

File: compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+- A domain name
```
Comment (RoRoJ) [inline compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx:19]:
```suggestion
- You have a domain name 
```

File: compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+- A running [Serverless Function](https://www.scaleway.com/en/docs/compute/functions/how-to/deploy-a-function/)
+- A domain name 
+
+</Message>
+
+1. Add a CNAME pointing to your function default URL from the [domains dashboard](/network/dns-cloud/how-to/manage-dns-records/).
```
Comment (RoRoJ) [inline compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx:23]:
```suggestion
1. Add a CNAME record pointing to your function's default URL from the [domains dashboard](/network/dns-cloud/how-to/manage-dns-records/).
```

Comment (RoRoJ) [reply compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx:23]: Does the domain DNS have to be managed by SCW?

File: compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+2. Click the **Endpoints** tab on your Functions' page.
```
Comment (RoRoJ) [inline compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx:24]:
```suggestion
2. Click the **Endpoints** tab on your Function's Overview page.
```

File: compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+  <Lightbox src="scaleway-functions-domain.png" alt="" />
+3. Click **+ Add a New Endpoint**.
+4. Enter your custom URL and validate the form.
```
Comment (RoRoJ) [inline compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx:27]:
```suggestion
4. Enter your custom domain URL and validate the form.
```

File: compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx (hunk 6 above)
Comment (bene2k1) [reply compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx]: I dont't think it has forcefully be managed by scw DNS - we can generalize this if you prefer

**Review by ldecarvalho-doc — APPROVED** [review]

**Review by ofranc — APPROVED** [review]

**Pull request merged**